src/Nri/Ui/Button/V10.elm

```diff
@@ -926,15 +932,51 @@ buttonStyles :
         , narrowMobileWidth : Maybe ButtonWidth
         , state : ButtonState
         , customStyles : List Style
+        , pressed : Maybe Bool
     }
-    -> Style
-buttonStyles ({ style, state, customStyles } as config) =
-    Css.batch
-        [ buttonStyle
-        , sizeStyle config
-        , colorStyle style state
-        , Css.batch customStyles
+    -> List Style
+buttonStyles ({ state, customStyles } as config) =
+    let
+        ( stringBoxShadow, pressedStyles, style ) =
+            case config.pressed of
+                Just True ->
+                    ( [ "inset 0 3px 0 " ++ ColorsExtra.toCssString pressedShadowColor ]
+                    , Css.batch
+                        [ Css.boxShadow5 Css.inset Css.zero (Css.px 3) Css.zero pressedShadowColor
+                        , Css.color Colors.navy
+                        , Css.backgroundColor Colors.glacier
+                        , Css.borderBottomWidth (Css.px 1)
+                        ]
+                    , secondaryColors
+                    )
+
+                Just False ->
+                    ( []
+                    , Css.hover [ Css.color Colors.navy ]
+                    , secondaryColors
```

Comment:
In future, `toggleButtonPressed` could possibly be made to play nicely with themes other than secondary (well, primary/danger/premium might be harder than the secondary/tertiary themes). But no need to do that now.

Reply:
Agreed! I was thinking about doing it, but then it seemed like since Design hadn't asked for it, it was unlikely to be worth it. (We also don't actually have very many toggle buttons in the app to start with! I think just formatting questions and grade selection)